scripts/postprocess.py

```diff
@@ -11,9 +11,33 @@
 
 sns.set_theme()
 
-# plt_type : ppn | time
-def _gen_nodes_plot(plt_type, title, ylabel, file_name, df, logx, logy=False, combine_mb=False, filts=None, replaces=None):
-    fig, ax = plt.subplots()
+
+def _gen_nodes_plot(
+    plt_type,
+    title,
+    df,
+    combine_mb=False,
+    filts=None,
+    replaces=None,
+    styles=None,
+    subplot_args=None,
+    fill_area=True,
+):
+    """
+    Args:
+        plt_type:       ppn | time
+        title:          title of the plot
+        df:             the pandas.DataFrame with the data for the plot
+        combine_mb:     group also by mb sizes
+        filts:          [regex] for matching benchmark names to filter from the plot
```

Comment:
- is `[regex]` a standard way to document an array containing one ore more regex? I'm used to fact that square brackets indicate optional arguments.
- I would interpret `filter from` as `matching are excluded from`. Am I wrong?
   Maybe rewriting as following is more clear:
   `benckmarks matching one of these regular expressions will appear in the plot (None: all benchmarks will appear).`

Reply:
Good points. I tried to address your comments. Please, give it a look 😉